modules/ROOT/pages/_partials/commons/common-vector-search.adoc

```diff
@@ -0,0 +1,219 @@
+
+Vector encoding reduces the size of the vectors index by algorithmic compression.
+You can configure the Vector Encoding in Couchbase Lite to address your application's needs.
+
+This vector encoding compression reduces disk space required and I/O time during indexing and queries, but greater compression can result in inaccurate results in distance calculations.
+
+Vector Search for Couchbase Lite supports the following encoding algorithms:
+
+* None - This will return the highest quality results but at high performance and disk space costs
+
+* Scalar Quantizer - This reduces the number of bits used for each number in a vector. 
+The number of bits per component can be set to 4, 6, or 8 bits.
+The default setting in Couchbase Lite is 8 bits Scalar Quantizer or SQ-8.
+
+* Product Quantizer - This reduces the number of dimensions and bits per dimension. 
+It splits the vectors into multiple subspaces and performing scalar quantization on each space independently before compression.
+This can produce higher quality results than Scalar Quantization at the cost of greater complexity.
+
+NOTE: Quantizers are algorithmic processes that map input values from a larger set to output values in a smaller set, common quantization processes can include operations such as rounding and truncation.
+
+=== About Centroids
+
+Centroids are vectors that function as the center point of a vector cluster within the data set.
+Each vector is then associated to the vector it is closest to by https://en.wikipedia.org/wiki/K-means_clustering[k-means clustering.]
+Each Centroid is contained within a bucket along with its associated vectors, the greater amount of Centroids, the greater the potential accuracy of the model.
+This will also incur a longer indexing time.
+
+Choosing Centroids in Vector Search involves trade-offs that can impact clustering effectiveness and search efficiency. The initial selection of Centroids, the number chosen, and their sensitivity to high dimensionality and outliers affect the quality of vector clustering.
+
+The general guideline for the amount of Centroids is approximately the square root of the number of documents.
+
+=== About Probes
+
+The number of Probes refers to the maximum number of Centroid buckets that the search algorithm will to check to look for similar vectors to a given query vector.
+You can change the number of Probes by altering the value of the `NumProbes` variable xref:{param-module}:working-with-vector-search.adoc#create-a-vector-index[shown in the following example].
+Couchbase recommends that when setting a custom number of probes, the number should be at least 8 or 0.5% the number of Centroids used.
+
+=== About Dimensions
+
+Vector dimensions describes the amount of numbers in a given vector embedding, commonly known as its' width.
+The greater the amount of dimensions, the greater accuracy of results, this also results in greater compute and memory costs and a increase in the latency of the search.
```

Comment:
Reword slightly, like the comment above about Centroids, to make it clear that we're talking the advantages and tradeoffs of increasing the number of dimensions. Something like

```suggestion
The greater the number of dimensions, the greater the accuracy of results.
However, a greater number of dimensions also results in greater compute and memory costs and a increase in the latency of the search.
```